pkg/metaserver/external/manager.go

```diff
@@ -14,8 +14,25 @@
 
 package external
 
-import "context"
+import (
+	"context"
 
+	"github.com/kubewharf/katalyst-core/pkg/util/cgroup/common"
+)
+
+// ExternalManager contains a set of managers that execute configurations beyond the OCI spec.
 type ExternalManager interface {
 	Run(ctx context.Context)
+
+	GetCgroupIDForContainer(podUID, containerID string) (uint64, error)
+	ListCgroupIDsForPod(podUID string) ([]uint64, error)
+
+	ApplyNetClass(podUID, containerId string, data *common.NetClsData) error
+	ClearNetClass(cgroupID uint64) error
+
+	CheckSupportRDT() (bool, error)
```

Comment:
why not just use NetworkManager and RDTManager interface here, instead of re-define the functions here

Reply:
done